### DISCUSSION TITLE
wind gusts data and other ideas / improvements

### DISCUSSION THREAD

**Discussion by jortaca** [open-meteo/open-meteo#912] [Ideas]
Hello Patrick, I am writing this post without enough knowledge, so I'm sorry in advance in case I say something stupid.

- Regarding the Arome HD gusts issue, you mentioned that you could finally recover it using the U/V vectors. Could this method be applied to other models that don’t have the 10m wind gusts data, such as Arpae Cosmo 2km, JMA, or ECMWF? Or do these models also lack that data?

- Have you considered adding the WRF Model? I heard that it is a good model for wind predictions. 
https://openskiron.org/en/openwrf (I don't know if this is the best source)

- And adding the basic Waves (m. period, direction) data to the Best Match? (in case it is a sea location)

- I think you have already explained that it is difficult, but it would be great to have information on the latest update time or when the next update is expected.

Thanks for your great API.

Jorge Tarazona

**Comment by patrick-zippenfenig** [comment]
Hi,

1. No, it is not possible to calculate wind gusts for other domain. Wind gusts need to be available either as U/V components or directly as speed. Some domains lack wind gust forecasts entirely.

2. I am aware of those models, however, the resolution at 12 km is fairly limited. Other domains use significantly higher resolutions.

3. Do you mean adding wave forecasts to the general weather forecast API? Servers for weather forecast and marine forecasts are separated instances. This is important to independently scale the Open-Meteo system to more nodes. Aggregating all data on a single node, will make scaling very complicated.

4. Yes, the last model init and update time will be exposed in the future. Currently I lack this information entirely in the system and I am also not sure which update times to display in case multiple models are combined.
